src/defs.h

```diff
@@ -9,7 +9,7 @@
 #define MAX_PARAMS 8
 #define MAX_LOCALS 48
 #define MAX_FIELDS 32
-#define MAX_FUNCS 1024
+#define MAX_FUNCS 2048
```

Comment:
Can you lower?

Reply:
Sure. The `func_tries_idx` almost reaches this limit before changing, so I just double the value of it.

Actually, the `funcs_idx` only uses low hundreds of it. I add another limitation to separate these two.